README.md

```diff
@@ -35,6 +47,63 @@ Nextflow requires some memory resources to be allocated for overhead - consequen
 
 ---
 
+## Running on AWS Batch
+
+NovelTree can be run on AWS Batch for large-scale analyses. To use AWS Batch:
+
+```bash
+nextflow run . \
+  -profile awsbatch \
+  --awsqueue <your-batch-queue> \
+  --awsregion <your-aws-region> \
+  -work-dir s3://<your-bucket>/work \
+  --outdir s3://<your-bucket>/results \
+  --input <input.csv>
+```
+
+**Requirements:**
+- AWS Batch compute environment and job queue must be configured
+- Work directory and output directory must be S3 buckets
+- Appropriate IAM permissions for Batch and S3 access
```

Comment:
Related to the above, probably worthwhile specifying that all of the workflow docker images also need to be built and hosted somewhere on a repository that can be accessed by the workflow?